jans-cedarling/cedarling/src/common/policy_store.rs

```diff
@@ -5,155 +5,430 @@
  * Copyright (c) 2024, Gluu, Inc.
  */
 
-use std::collections::HashMap;
-
 use super::cedar_schema::CedarSchema;
+use base64::prelude::*;
+use cedar_policy::PolicyId;
+use serde::{de::Visitor, Deserialize, Deserializer};
+use std::{collections::HashMap, fmt};
 
-/// PolicyStoreMap it is a store for `PolicyStore` accessible by key.
-#[derive(Debug, serde::Deserialize)]
-pub(crate) struct PolicyStoreMap {
-    #[serde(flatten)]
-    pub policy_stores: HashMap<String, PolicyStore>,
-}
-
-/// PolicyStore contains all the data the Cedarling needs to verify JWT tokens and evaluate policies
+/// Represents the store of policies used for JWT validation and policy evaluation in Cedarling.
+///
+/// The `PolicyStore` contains the schema and a set of policies encoded in base64,
+/// which are parsed during deserialization.
 #[derive(Debug, Clone, serde::Deserialize)]
 pub struct PolicyStore {
-    pub schema: CedarSchema,
-    #[serde(deserialize_with = "deserialize_policy_set::parse")]
-    pub policies: cedar_policy::PolicySet,
+    /// The cedar version to use when parsing the schema and policies.
+    #[serde(deserialize_with = "check_cedar_version")]
+    #[allow(dead_code)]
+    pub cedar_version: String,
+
+    /// Cedar schema in base64-encoded string format.
```

Comment:
It is unclear why it is `base64-encoded string format` but 
`pub cedar_schema: CedarSchema,` is not string.

Reply:
fixed in: [208596a](https://github.com/JanssenProject/jans/pull/9910/commits/208596aa715bb0b9bcfee92fc76f87e7ebdb765a)